src/ast/mod.rs

```diff
@@ -223,6 +223,13 @@ pub enum Expr {
     /// A parenthesized subquery `(SELECT ...)`, used in expression like
     /// `SELECT (subquery) AS x` or `WHERE (subquery) = x`
     Subquery(Box<Query>),
+    /// A parameter declaration.
```

Comment:
I think calling it "parameter declaration" elsewhere (i.e. the variant name and the parser methods) too would be preferred. Otherwise we end up with parse_optional_{args/parameters}, though arguments and parameters are synonymous in my understanding.

Reply:
Makes sense. I will move this to a separate struct